webapp/portlet/src/main/webapp/share-activity/components/ShareActivityDrawer.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <exo-drawer
+    ref="shareActivityDrawer"
+    body-classes="hide-scroll decrease-z-index-more"
+    right
+    @closed="close">
+    <template slot="title">
+      {{ $t('UIActivity.share.drawer.popupTitle') }}
+    </template>
+    <template slot="content">
+      <v-form
+        v-if="activityId"
+        ref="activityShareFrom"
+        class="flex news-form share-activity"
+        flat>
+        <div class="d-flex flex-column flex-grow-1">
+          <div class="d-flex flex-row">
+            <span class="mt-4 ml-2 mb-2">{{ $t('UIActivity.share.shareInSpaces') }} </span>
+          </div>
+          <div class="d-flex flex-row flex-grow-1 activitySpaceSuggester">
+            <share-activity-suggester :spaces="spaces" class="ml-2" />
+          </div>
+          <div class="d-flex flex-row">
+            <textarea
+              v-model="description"
+              :placeholder="$t('UIActivity.share.sharedActivityPlaceholder')"
+              class="ml-2 ignore-vuetify-classes activityShareDescription">
+              </textarea>
+          </div>
+        </div>
+      </v-form>
+    </template>
+    <template slot="footer">
+      <div class="d-flex justify-end">
+        <v-btn
+          :disabled="shareDisabled"
+          class="btn btn-primary mr-2"
+          @click="shareActivity">
+          {{ $t('news.share.share') }}
+        </v-btn>
+      </div>
+    </template>
+  </exo-drawer>
+</template>
+
+<script>
+export default {
+  props: {
+    activityId: {
+      type: String,
+      default: ''
+    },
+    activityType: {
+      type: String,
+      default: ''
+    }
+  },
+  data: () => ({
+    showShareNewsDrawer: false,
```

Comment:
```suggestion
    showShareActivityDrawer: false,
```
to be removed if not used